src/validation/obfl.rng

```diff
@@ -32,9 +33,15 @@
     <optional>
       <attribute name="hyphenate">
         <choice>
-          <value>true</value>
-          <value>false</value>
-          <value></value>
+          <text/>
+          <ref name="leader"/>
+          <ref name="marker"/>
+          <ref name="br"/>
+          <ref name="evaluate"/>
+          <ref name="page-number"/>
+          <ref name="marker-reference"/>
+          <ref name="span"/>
+          <ref name="style"/>
```

Comment:
I don't know what happened here, but this change is certainly not correct and it causes that the RNG becomes invalid. As a result, the `buildSite` command fails.